app/models/drug_stock.rb

```diff
@@ -0,0 +1,9 @@
+class DrugStock < ApplicationRecord
+  belongs_to :facility
+  belongs_to :user
+  belongs_to :protocol_drug
+
+  validates :in_stock, numericality: true, presence: true
+  validates :received, numericality: true, presence: true
```

Comment:
Can we sanity check whether `received` can be empty? I think it might be possible. Only `in_stock` is required for the basic reporting we want to do. `received` lets us calculate consumption later, so I reckon people might wanna leave it blank to start, at least sometimes.

Reply:
Shouldn't they enter 0 then?

Reply:
I've made the change for now, can add the check if we decide to later

Reply:
Coolbeans. Yeah `0` and `null` should be treated differently. `0` means "nothing was received", `null` means "I don't know".